.jshintrc

```diff
@@ -31,7 +31,7 @@
   "undef"         : true,     // Require all non-global variables be declared before they are used.
   "unused"        : true,     // Warn when variables are created but not used.
   "trailing"      : true,     // Prohibit trailing whitespaces.
-  "es3"           : true,     // Prohibit trailing commas for old IE 
+  "es3"           : false,     // Prohibit trailing commas for old IE 
```

Comment:
It seems like this should just be fixed in this PR then :-/ Leaking it to master seems to provide very little long/mid-term value, though I 100% understand the short-term aid during dev.